[2024-05-19T20:35:05.053+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-19T20:35:00.637835+00:00 [queued]>
[2024-05-19T20:35:05.061+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-19T20:35:00.637835+00:00 [queued]>
[2024-05-19T20:35:05.062+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-05-19T20:35:05.079+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): seceond_task> on 2024-05-19 20:35:00.637835+00:00
[2024-05-19T20:35:05.087+0000] {standard_task_runner.py:60} INFO - Started process 482 to run task
[2024-05-19T20:35:05.090+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'seceond_task', 'manual__2024-05-19T20:35:00.637835+00:00', '--job-id', '415', '--raw', '--subdir', 'DAGS_FOLDER/frist-dag.py', '--cfg-path', '/tmp/tmpovewhcbj']
[2024-05-19T20:35:05.091+0000] {standard_task_runner.py:88} INFO - Job 415: Subtask seceond_task
[2024-05-19T20:35:05.137+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-19T20:35:00.637835+00:00 [running]> on host 7b23a6949395
[2024-05-19T20:35:05.216+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='seceond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T20:35:00.637835+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-19T20:35:00.637835+00:00'
[2024-05-19T20:35:05.217+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-19T20:35:05.219+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'wc -l /opt/***/output.csv']
[2024-05-19T20:35:05.232+0000] {subprocess.py:86} INFO - Output:
[2024-05-19T20:35:05.238+0000] {subprocess.py:93} INFO - 240019 /opt/***/output.csv
[2024-05-19T20:35:05.239+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-19T20:35:05.262+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=seceond_task, execution_date=20240519T203500, start_date=20240519T203505, end_date=20240519T203505
[2024-05-19T20:35:05.303+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-19T20:35:05.321+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
